src/comm/diagnostics_codes_table.h

```diff
@@ -0,0 +1,86 @@
+//
```

Comment:
The license on the new files is wrong - should be our enway license

Reply:
Changed.